pmm-app/src/pmm-add-instance/components/AddRemoteInstance/FormParts/FormParts.types.ts

```diff
@@ -8,6 +8,7 @@ export interface AdditionalOptionsFormPartProps {
   instanceType?: string;
   loading: boolean;
   remoteInstanceCredentials: RemoteInstanceCredentials;
+  form: any;
```

Comment:
You can use FormApi from final-form here.

Reply:
done